manifests/config.pp

```diff
@@ -13,7 +13,7 @@
           owner   => 'root',
           group   => 'root',
           mode    => '0444',
-          content => epp('jmxtrans/jmxtrans.service.epp', { 'user' => $::jmxtrans::user }),
+          content => epp($::jmxtrans::systemd_template, { 'user' => $::jmxtrans::user }),
```

Comment:
This should be modified to pass in the additional variables you use as parameters rather than having the template reference namespaced variables.